src/api/mock-interceptor.ts

```diff
@@ -94,7 +94,7 @@ export const interceptRequestsOnMock = () => {
   mock.onGet(new RegExp(`${PublicationsApiPaths.FOR_APPROVAL}/*`)).reply(200, mockMyPublications);
```

Comment:
Prøvde å opprette publikasjon med DOI (https://doi.org/10.1126/science.169.3946.635), endret tittel, men får dette på PUT:
```
{
  "title" : "Bad Gateway",
  "status" : 502,
  "detail" : "{\"message\": \"Internal server error\"}"
}
```
virker det hos deg?

Reply:
det virker hvis jeg setter date også. Så da er det nok noe feil med mapping der. det var jeg forsåvidt klar over også, siden man må navigere til en annen fane så tilbake til description for at datofeltet skal bli satt

Reply:
posten blir oppdatert, men det er noe galt i backend. @janbj ser på det når han får tid.